rbac/server/api/packs.py

```diff
@@ -127,10 +128,40 @@
     return json({"roles": request.json.get("roles")})
 
 
+@PACKS_BP.delete("api/packs/<pack_id>")
+@authorized()
+async def delete_pack(request, pack_id):
+    """Delete pack from NEXT
+    Args:
+        request:
+            object: request object
+        pack_id:
+            str: ID of pack to delete
+    """
+    txn_key, txn_user_id = await utils.get_transactor_key(request)
+
+    pack = await packs_query.check_pack_by_pack_id(request.app.config.DB_CONN, pack_id)
+    if not pack:
+        raise ApiBadRequest(
+            "Error: Pack does not currently exist or has already been deleted."
+        )
+    owners = await packs_query.get_pack_owners_by_id(
```

Comment:
You have checked for owners.  You have not used the utility function: utils.check_admin_status(txn_user_id).  They must be a NEXT admin or an owner to be able to do this.  The utility function will return a boolean if the request maker is a next admin
line 151 needs to use this function to determine if they are a NEXT admin.  Which is different then being in the owners payload.